docs-v2/reference/sdks/node.mdx

```diff
@@ -487,6 +531,50 @@ await nango.updateMetadata('<INTEGRATION-ID>', 'CONNECTION-ID', { 'CUSTOM_KEY1':
 **Response**
 
 Empty response.
+<Expandable>
+```json
+{
+    "CUSTOM_KEY1": "CUSTOM_VALUE1"
+}
```

Comment:
I don't get why we should answer the metadata?

Reply:
Serves as confirmation of successful receipt